src/theme/DocItem/index.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, {useEffect} from 'react';
```

Comment:
Before, when navigating to a new page, we wouldn't scroll to the top, so we would just stay at the bottom or the middle